AlphaWallet/Sell/ViewControllers/EnterSellTokensCardPriceQuantityViewController.swift

```diff
@@ -49,7 +49,7 @@ class EnterSellTokensCardPriceQuantityViewController: UIViewController, TokenVer
         return viewModel.token.server
     }
     let assetDefinitionStore: AssetDefinitionStore
-    lazy var pricePerTokenField = AmountTextField(tokenObject: viewModel.token)
```

Comment:
I think the fix might be simpler and more correct by replacing:

```
AmountTextField(tokenObject: viewModel.token)
```

with:

```
AmountTextField(tokenObject: ethToken)
```

And you can get ethToken via `TokensDataStore(forServer:)`.

Can you see if that makes sense and is right? The problem being in the send screen, we want the amount textfield for entering the amount of token the send screen is for. But in this screen, no matter which token it is, we want the amount textfield to be for entering Ether (or whichever native crypto token like xDai, etc).